appinfo/info.xml

```diff
@@ -30,7 +30,7 @@
 	<screenshot>https://user-images.githubusercontent.com/1374172/79554966-278e1600-809f-11ea-82ea-7a0d72a2704f.png</screenshot>
 	<dependencies>
 		<php min-version="8.0" max-version="8.2" />
-		<nextcloud min-version="26" max-version="27" />
+		<nextcloud min-version="27" max-version="28" />
```

Comment:
now that im thinking, should i change the min-version to 27? or should i leave it 26?

Reply:
Leave 26 until it EOLs or we need a 27 API